src/dependencies.jl

```diff
@@ -1043,13 +1043,8 @@ using SHA
 function sha_check(path, sha)
     open(path) do f
         calc_sha = sha256(f)
-	# Workaround for SHA.jl API change.  Safe to remove once SHA versions
-	# < v0.2.0 are rare, e.g. when Julia v0.4 is deprecated.
```

Comment:
It won't be a problem since version's less thatn .3 don't have 0.5 support.
https://github.com/staticfloat/SHA.jl/blob/v0.2.2/REQUIRE